### DISCUSSION TITLE
[Solved] Anyone having issues with BindableProperty?

### DISCUSSION THREAD

**Discussion by BinaryAssault** [dotnet/maui#5503] [Q&A] [answered]
I have a custom control, it's only 2 buttons and an entry, all in xaml with standard Maui controls. The control exposes a single BindableProperty, call it Test that is bound to the text property of the entry in the custom control. I have a view that implements the custom control and maps Test to a value in the view model. 

One of the buttons on the custom control simply issues Test++
Stepping through the debugger, all my bindings in the custom control are working and issuing Test++ is updating the BindableProperty.

But I can't get them to propagate back into the view that implants the custom control. And the view model isn't updating either.

**Comment by yyliveyy** [comment]
Did you implement the INotifyPropertyChanged interface?

**Reply by BinaryAssault** [reply]
I'm not infront of the code, but I think I was using the community toolkit mvvm package with OnPropertyChanged(). I'll double check tomorrow

**Reply by BinaryAssault** [reply]
I added INotifyPropertyChanged and no change. I also looked over how an entry uses Bindable properties and mimicked it and had no luck.

**Comment by nextcodelab** [comment]
Like this maybe...
[https://github.com/dotnet/maui/issues/5381](https://github.com/dotnet/maui/issues/5381)

**Reply by BinaryAssault** [reply]
Not sure that is the same issue. I thought I solved mine, but it's still present. The view where my custom control is isn't updating the VM on change.

**Comment by BinaryAssault** [comment, marked as answer]
Turns out I just didn't set up my XAML correctly. All working now. Gerald Versluis video is what ended up leading me to a functional control.

https://www.youtube.com/watch?v=cbmoaD-PDtA
